Add photo grid component(s)

suneettipirneni opened the pull request:
Adds photo grid component closes #62, use `/main` on the AWS preview to see visual changes.

suneettipirneni committed:
Initial Commit

suneettipirneni committed:
remove box shadow

suneettipirneni committed:
Add animations

suneettipirneni committed:
animation pt. 2

suneettipirneni committed:
Add floating button

ctcuff commented on src/scss/floating-button.scss:
```diff
@@ -0,0 +1,25 @@
+    justify-content: center;
+    bottom: 50px;
+    right: 80px;
+    height: 60px;
```
Just an opinion but I think `bottom" 24px` and `right: 60px` is a better a fit

ctcuff reviewed (approved):
Looks amazing 👍 Another opinion: I think we can remove `margin: 20px;` in `main-page.scss`. I think that would give the image content a bit more breathing room and it puts the scrollbar to the very right of the page which feels a bit more natural.

Also, this isn't related to this PR but can you remove
```scss
@supports (backdrop-filter: blur(1px)) {
  background-color: rgba(255, 255, 255, 0.7);
    backdrop-filter: blur(15px);
}
```
in `navbar.scss`? I realize now that the main content will never scroll under it so the effect is unnecessary now

suneettipirneni committed:
Add antd images

deepsource-autofix[bot] committed:
Autofix issues in 2 files

Resolved issues in the following files via DeepSource Autofix:
1. src/index.jsx
2. src/pages/MainPage.jsx

suneettipirneni committed:
Fix button issues

suneettipirneni committed:
Merge branch 'design/photogrid' of https://github.com/Imageus-OSS/client into design/photogrid

suneettipirneni committed:
Fix linting issue

suneettipirneni committed:
Merge branch 'main' of https://github.com/Imageus-OSS/client into design/photogrid

suneettipirneni committed:
make requested change

suneettipirneni committed:
remove backdrop filter on nav
